### PR TITLE
enable platform selector for feed pages (bug 1127080)

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: src/media/js/compat_filter.js [hunk 1]
```diff
@@ -37,9 +37,7 @@ define('compat_filter',
     var featureProfile = utils.getVars().pro || '7fffffffffff0.51.6';
 
     // Don't do device filtering for these endpoints.
-    var ENDPOINT_DEVICE_FILTER_REDLIST = [
-        'feed', 'feed-app', 'feed-brand', 'feed-collection'
-    ];
+    var ENDPOINT_DEVICE_FILTER_REDLIST = ['feed-app'];
```
Comment (mstriemer) [inline src/media/js/compat_filter.js]: Is this redlist to avoid blacklist? `EXCLUDE_DEVICE_FILTER_ENDPOINTS` maybe?

**Comment by mstriemer** [comment]
r+

File: src/media/js/compat_filter.js (hunk 1 above)
Comment (ngokevin) [inline src/media/js/compat_filter.js]: yeah, i think it's a bit silly to try to be PC with variable names...so i just made it red :P

**Pull request merged**